alonzo/formal-spec/protocol-parameters.tex

```diff
@@ -6,29 +6,42 @@ \section{Language Versions and Cost Models}
 
 \vspace{12pt}
 \begin{tabular}{lp{5in}}
-  $\Language$ &
-  This represents the language name/tag, including the
-  version number.
-  \\
-  $\ExUnits$ &
-  A term of this type contains two integer values,
-  $(mem, steps)$.
-  These represent abstract notions of the relative memory usage and script execution steps,
-  respectively.
-  \\
   $\CostMod$ &
   A term of this type represents the vector of coefficients that are used to generate
   a term of type $\ExUnits$ given a vector of some resource primitives. The mapping is defined
   concretely by the specific version of the Plutus interpreter that is associated with $\Language$.
   We keep this type as abstract in the specification.
+  \begin{note}
+    Add a link to $\CostMod$ documentation
```

Comment:
I think we shouldn't add TODO's in the repo at this point, we probably want this document to be presentable to the public as much as possible